var/spack/repos/builtin/packages/py-h5io/package.py

```diff
@@ -9,7 +9,7 @@
 class PyH5io(PythonPackage):
     """Python Objects Onto HDF5."""
 
-    homepage = "http://h5io.github.io"
+    homepage = "https://h5io.github.io"
```

Comment:
I cannot confirm this URL. When I attempt to access it though a browser, I get `There isn't a GitHub Pages site here.`.

Reply:
Let's go with plain github then.
```suggestion
    homepage = "https://github.com/h5io/h5io"
```